packages/common-server/src/parser.ts

```diff
@@ -168,6 +168,7 @@ export class SchemaParserV2 extends ParserBaseV2 {
         });
       }
 
+      ent.isIdAutoGenerated = true;
```

Comment:
should the type be updated to `SchemaRaw`?

Reply:
Added type SchemaRaw